app/models/review_bid.rb

```diff
@@ -7,7 +7,7 @@ class ReviewBid < ApplicationRecord
   # returns the bidding data needed for the assigning algorithm
   # student_ids, topic_ids, student_preferences, topic_preferences, max reviews allowed
 
-  def self.bidding_data(assignment_id, reviewer_ids)
+  def bidding_data(assignment_id, reviewer_ids)
```

Comment:
If this is an instance method, it should relate to a specific review_bid.  The code makes it look like it relates to more than a single bid, though I have a hard time seeing what the code is actually doing.

Reply:
Also, there certainly needs to be a method comment explaining what the method does.